frontend/src/screens/History/History.tsx

```diff
@@ -3,38 +3,12 @@ import Header from "../../components/Header/Header";
 import SessionHistory from "../../components/SessionHistory/SessionHistory";
 import styles from "./History.module.css";
 
-// TODO:  Update all types
-type Exercise = {
-  id: number;
-  name: string;
-  videoUrl: string;
-  workoutTemplateExercises: any;
-  createdAt: Date;
-};
-
-type SetRecord = {
-  id: number;
-  workoutSessionId: number;
-  workoutSession: any;
-  exerciseId: number;
-  exercise: Exercise;
-  reps: number;
-  weight: number;
-  createdAt: Date;
-};
-
-type Session = {
-  date: Date;
-  workoutTemplate: any;
-  duration: number; // in seconds
-  completionStatus: "COMPLETED" | "PARTIAL" | null;
-  setRecords: SetRecord[];
-};
-
-
 const History = () => {
+  const [totalTimeWorkingout, setTotalTimeWorkingout] = useState(150);
```

Comment:
should we set these default values to be 0? assuming all users start with no workouts?

Reply:
Yes, it should be 0. I had the value set because I wanted to take a screenshot of what I want the screen to look like. That should've been changed back to 0.